src/main/java/io/vertx/proton/impl/ProtonConnectionImpl.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 import java.util.UUID;
 
+import io.vertx.proton.*;
```

Comment:
Agree on that I have to check if it's possible to disable such feature in IntelliJ IDEA